feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -1335,7 +1335,11 @@ bool target_vif_config_del(const struct schema_Wifi_VIF_Config *vconf)
 		if (strcmp(s->type, "wifi-iface")) continue;
 
 		ifname = uci_lookup_option_string( vif_ctx, s, "ifname" );
-		if (!strcmp(ifname,vconf->if_name)) {
+		if (ifname == NULL) {
```

Comment:
Seems like we're duplicating the `uci_section_del` call, may as well merge the `if` and `else if` statements

```suggestion
		if (ifname == NULL || !strcmp(ifname,vconf->if_name)) {
```

Reply:
I only want the break in the 2nd case.